### DISCUSSION TITLE
Video DowloadHelper falhou

### DISCUSSION THREAD

**Discussion by hugodesen4** [aclap-dev/video-downloadhelper#724] [Help and Q&A]
### Your question

Olá, como vão?
Estava utilizando o VideodowloadHelper e falhou. Não me parece que seja o formato do vídeo, uma vez que já baixei vídeos do mesmo site.
A mensagem que apareceu foi:

Download falhou
Infelizmente, falhamos no download dessa mídia específica. Tentamos oferecer suporte ao maior número possível de sites, então ajudaria muito se você pudesse relatar esse erro!

Apareceu um código, espero que ajude os colaboradores:

Detalhes do bug:

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://play.ocorpoexplica.com.br/assistir/1572/papo-saudavel-08022022",
  "error": {
    "details": "SideDownload error: : [vost#0:0/copy @ 000001e62a9e6040] Error submitting a packet to the muxer: No space left on device\r\n[out#0/mp4 @ 000001e62c7069c0] Error muxing a packet\r\n[out#0/mp4 @ 000001e62c7069c0] Error writing trailer: No space left on device\r\n[out#0/mp4 @ 000001e62c7069c0] Error closing file: No space left on device\r\n",
    "downloadable_id": "downloadable_6159760088082142",
    "error": "coapp_failure",
    "id": "coapp_failure_13f92db6-f92e-4ea2-ad4d-4038be3260e8"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://play.ocorpoexplica.com.br/assistir/1572/papo-saudavel-08022022",
  "error": {
    "details": "SideDownload error: : [vost#0:0/copy @ 000001e62a9e6040] Error submitting a packet to the muxer: No space left on device\r\n[out#0/mp4 @ 000001e62c7069c0] Error muxing a packet\r\n[out#0/mp4 @ 000001e62c7069c0] Error writing trailer: No space left on device\r\n[out#0/mp4 @ 000001e62c7069c0] Error closing file: No space left on device\r\n",
    "downloadable_id": "downloadable_6159760088082142",
    "error": "coapp_failure",
    "id": "coapp_failure_13f92db6-f92e-4ea2-ad4d-4038be3260e8"
  }
}

**Comment by Mattmjs** [comment]
No space left on device ? Which disk drive are you using ?
